src/client/js/controller/index.js

```diff
@@ -459,11 +459,11 @@ class Controller extends EventEmitter {
     model.clear();
 
     // Load parameters or set default values.
-    let database = state['database'] || 'head';
+    let database = state['database'] || 'complete';
     let allDatasets = DataService.getDatasetList(database);
     let datasets = intersection(allDatasets, state['datasets'] || []);
     if (datasets.length === 0) {
-      datasets = (database == 'head') ? ['randi_funconn_wildty'] : allDatasets;
+      datasets = (database == 'complete') ? ['randi_funconn_wildcp'] : allDatasets;
```

Comment:
Would this make more sense?
```
      datasets = (database == 'head') ? ['randi_funconn_wildty'] : ((database == 'complete') ? ['randi_funconn_wildcp'] : allDatasets);
```

Reply:
That's a good point Rob. I'm not sure what we ultimately want, but I like the idea of making the Randi et al datasets the single default for each group.

However, I'm not actually sure how to do that, since this line only executes when there are no previously defined datasets. Once you hit the page for the first time, there is a `datasets.length` is no longer zero. You end up getting all datasets in the `head` group if you switch. I'll see if I can find a simple way to achieve that, but if not, I'll merge this and create a new issue.

Reply:
Regarding our lunchtime discussion, doesn't this line mean that the length of the datasets array depends on the state of the selections:
```
let datasets = intersection(allDatasets, state['datasets'] || []);
```

Reply:
Well, I tried for a bit to get this to work, but didn't have any luck. With this PR, the behavior is to start with Complete animal, Randi et al 2022 dataset only. For the other groups, all datasets are select. After the first visit, it uses whatever datasets ere last selected.

I'm happy to change this if we figure it out, but I'm suggesting we leave it as is for now.